guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -466,14 +479,14 @@ if option architecture = 00:06 {
 } elsif option architecture = 00:09 {
   filename "grub2/shim.efi";
 } elsif exists user-class and option user-class = "iPXE" {
-  filename "http://_satellite.example.com_/unattended/iPXE";
+  filename "http://_{foreman.example.com}_/unattended/iPXE";
 } else {
   filename "pxelinux.0";
 }
 ----
 +
 [NOTE]
-For `http://satellite.example.com/unattended/iPXE`, you can also use a {ProjectName} {SmartProxy} `http://capsule.example.comf:8000/unattended/iPXE`.
+For `http://{foreman.example.com}/unattended/iPXE`, you can also use a {ProjectName} {SmartProxy} `http://{smartproxy.example.com}:8000/unattended/iPXE`.
```

Comment:
Port variable for 8000 if Ewoud confirms, it is possible that both scenarios use 8000 and I am wrong, then don't touch it please.